app/config/storage.js

```diff
@@ -0,0 +1,37 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  connectionStr: Joi.string().when('useConnectionStr', { is: true, then: Joi.required(), otherwise: Joi.allow('').optional() }),
+  storageAccount: Joi.string().required(),
+  dpsContainer: Joi.string().required(),
+  daxContainer: Joi.string().required(),
+  inboundFolder: Joi.string().required(),
+  outboundFolder: Joi.string().required(),
+  archiveFolder: Joi.string().required(),
+  quarantineFolder: Joi.string().required(),
+  useConnectionStr: Joi.boolean().default(false),
+  createContainers: Joi.boolean().default(false)
+})
+
+const config = {
+  connectionStr: process.env.AZURE_STORAGE_CONNECTION_STRING,
+  storageAccount: process.env.AZURE_STORAGE_ACCOUNT_NAME,
+  dpsContainer: 'dps',
```

Comment:
This should be `batch`.

The `ffc-pay-gateway` will copy all files, payment and DPS to the `batch/inbound` directory.

`ffc-pay-batch-processor` will pick up any payment files, this service will pick up any DPS files based on filemasks.

`dps` container doesn't exist in any environment.  We could create one through the Platform repo, but doing that plus the redevelopment needed in `ffc-pay-gateway` to end up with two inbound locations for batch files doesn't feel right to me.